packages/playground/src/views/settings.vue

```diff
@@ -0,0 +1,245 @@
+                    v-model="newPassword"
+                    v-bind="{ ...passwordInputProps, ...validationProps }"
+                    autocomplete="off"
+                    class="pa-3"
+                  />
+                </div>
+              </template>
+            </v-tooltip>
+          </InputValidator>
+        </PasswordInputWrapper>
+        <PasswordInputWrapper #="{ props: confirmPasswordInputProps }">
+          <InputValidator
+            :value="confirmPassword"
+            :rules="[validators.required('A confirmation password is required.'), validateConfirmPassword]"
+            #="{ props: validationProps }"
+            ref="confirmPasswordInput"
+          >
+            <VTextField
+              label="Confirm Password"
+              v-model="confirmPassword"
+              v-bind="{
+                ...confirmPasswordInputProps,
+                ...validationProps,
+              }"
+              autocomplete="off"
+              class="pa-3"
+            />
+          </InputValidator>
+        </PasswordInputWrapper>
+      </form-validator>
+      <v-card-actions class="justify-end">
+        <v-btn :disabled="!isValidPassword" @click="UpdatePassword" class="justify-end ml-auto"
+          >Update</v-btn
+        ></v-card-actions
+      >
+    </v-card>
+    <v-card class="my-5"
+      ><v-card-title>Timeout</v-card-title>
+
+      <v-tooltip text="Set desired session timeout in seconds">
+        <template v-slot:activator="{ props }">
+          <v-card-text v-bind="props">Adjust Timeout <v-icon icon="mdi-information-outline" /></v-card-text>
+        </template>
+      </v-tooltip>
+      <form-validator v-model="isValidTimeout" valid-on-init>
+        <input-validator
+          :value="selectedTimeout"
+          :rules="[
+            validators.required('Timeout is required.'),
+            validators.isInt('Timeout must be a valid integer.'),
+            validateTimeout,
+          ]"
+          #="{ props }"
+          ref="timeoutInput"
+        >
+          <v-text-field
+            label="Enter timeout (sec)"
+            class="pa-5"
+            v-bind="props"
+            type="number"
+            v-model="selectedTimeout"
+          ></v-text-field>
+        </input-validator>
+        <v-card-actions class="justify-end">
+          <v-btn :disabled="!isValidTimeout" @click="UpdateTimeout" class="justify-end ml-auto"
+            >Update</v-btn
+          ></v-card-actions
+        >
+      </form-validator>
+    </v-card>
+  </view-layout>
+</template>
+<script lang="ts">
+import md5 from "md5";
+import { ref, watch } from "vue";
+import { useTheme } from "vuetify";
+
+import { createCustomToast, ToastType } from "@/utils/custom_toast";
+
+import { getCredentials, updateCredentials } from "../utils/credentials";
+export default {
+  name: "Settings",
+  setup() {
+    const theme = useTheme();
+    const KEY = "APP_CURRENT_THEME";
+
+    const themes = ["Dark Mode", "Light Mode"];
```

Comment:
As discussed in the group, we have three options for setting the display mode:

1. **System Mode**: Follows the system's default settings.
2. **Dark Mode**: Enables a dark-themed interface.
3. **Light Mode**: Enables a light-themed interface.

![Display Mode Options](https://github.com/user-attachments/assets/606f51cc-85d2-43b5-b959-b59b304d5ce7)